js/background.js

```diff
@@ -77,7 +77,8 @@
     theme: 'sunburst',
     font: 'Inconsolata',
     fontSize: 'medium',
-    lineNumbers: true
+    lineNumbers: true,
+    languageBlacklist: ''
```

Comment:
With the current way options are handled, I'm not sure how to create that nested structure.